fonts.css

```diff
@@ -1,24 +1,12 @@
 @font-face {
-    font-family: 'pt_serifbold';
+    font-family: 'PTSerif';
     src: url('fonts/ptserif-bold.woff') format('woff');
-    font-weight: normal;
+    font-weight: 700;
     font-style: normal;
 }
 @font-face {
-    font-family: 'pt_serifbold_italic';
-    src: url('fonts/ptserif-bolditalic.woff') format('woff');
-    font-weight: normal;
-    font-style: normal; 
-}
-@font-face {
-    font-family: 'pt_serifitalic';
-    src: url('fonts/ptserif-italic.woff') format('woff');
-    font-weight: normal;
-    font-style: normal;
-}
-@font-face {
-    font-family: 'pt_serif';
+    font-family: 'PTSerif';
     src: url('fonts/ptserif-regular.woff') format('woff');
-    font-weight: normal;
+    font-weight: 400;
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/HTML_CSS_2.0-Artem1998/pullRequest?prid=6242157)